### DISCUSSION TITLE
HRRR lat/long Projection Offset?

### DISCUSSION THREAD

**Discussion by gbamforth-bc** [blaylockbk/Herbie#220] [Q&A]
I'm using Herbie to download HRRR hourly precip data to overlay on an area of interest in ArcGIS. Currently I'm exporting a Herbie DataArray to Dataframe, then exporting that to csv with lat/long columns. When I import the table to ArcGIS, it looks like the entire layer is shifted East, despite the lat/long coordinates being unchanged. (See image below).
![image](https://github.com/blaylockbk/Herbie/assets/140541511/0fc2b565-0dfc-42e5-908e-ea1b80bf982d)

I know this is a crs issue, but am having difficulty with changing or finding the correct crs to use. Here is the code I'm running:

#Create a Herbie object for a 1-hr forecast:
H_1hr = herbie.Herbie(
    "2023-07-12 16:00",  # model run date
    model="hrrr",  # model name
    product="sfc",  # model produce name (model dependent)
    fxx=1,  # forecast lead time
)

#Extract the precipitation "layer" from the Herbie() object
ds_1hr = H_1hr.xarray(":APCP:",remove_grib=False)
df=ds_1hr.to_dataframe().reset_index()
df['longitude']=180-df['longitude']
df= df[["longitude","latitude","tp"]]
df.to_csv('path/to/file.csv')

**Comment by williamhobbs** [comment]
I'm not sure if this is helpful (or timely enough to be useful), but maybe there's something in this discussion that would give you a hint: https://github.com/blaylockbk/Herbie/discussions/45
